tests/ui_tests/app/page/common/table/page.py

```diff
@@ -111,3 +108,13 @@ def check_pagination(self, second_page_item_amount: int):
         assert (
             self.row_count == params["fist_page_cluster_amount"]
         ), f"Previous page should contains {params['fist_page_cluster_amount']} items"
+
+    @staticmethod
+    def __assert_enough_rows(required_row_num: int, row_count: int):
```

Comment:
the same in base_page.py. maybe remove it to common funcs?

Reply:
Yes, moved to `utils` for now